etc/kayobe/stackhpc-overcloud-dib.yml

```diff
@@ -37,7 +37,9 @@ stackhpc_overcloud_dib_elements:
 stackhpc_overcloud_dib_env_vars:
   DIB_BLOCK_DEVICE_CONFIG: "{{ stackhpc_overcloud_dib_block_device_config_uefi_lvm }}"
   DIB_BOOTLOADER_DEFAULT_CMDLINE: "nofb nomodeset gfxpayload=text net.ifnames=1 rd.auto"
-  DIB_CLOUD_INIT_DATASOURCES: "ConfigDrive"
+  DIB_GRUB_TIMEOUT: "5"
+  DIB_GRUB_TIMEOUT_STYLE: "countdown"
```

Comment:
That's the default (well, the default is hidden, but the docs say countdown is an alias) - do we need to set that?

Reply:
The ones that we've built before had DIB_GRUB_TIMEOUT set to 1 and DIB_GRUB_TIMEOUT_STYLE set to hidden.

Reply:
I asked about DIB_GRUB_TIMEOUT_STYLE, because DIB bootloader element  readme says:

* ``DIB_GRUB_TIMEOUT_STYLE`` sets the visibility of the ``grub`` menu.
  It defaults to ``hidden`` (or ``countdown`` as an alias). Set this to
  ``menu`` to display the menu and then wait for the timeout set by
  ``DIB_GRUB_TIMEOUT`` to expire before booting the default entry.
  
  That makes me think hidden is an alias for countdown - which is the default.

Reply:
It's true that in both cases Grub will wait for the timer to expire, however countdown will provide additional visual cue by displaying the countdown timer. Here's the definition from gnu.org:
```
‘GRUB_TIMEOUT_STYLE’
If this option is unset or set to ‘menu’, then GRUB will display the menu and then wait for the timeout set by ‘GRUB_TIMEOUT’ to expire before booting the default entry. Pressing a key interrupts the timeout.
If this option is set to ‘countdown’ or ‘hidden’, then, before displaying the menu, GRUB will wait for the timeout set by ‘GRUB_TIMEOUT’ to expire. If ESC or F4 are pressed, or SHIFT is held down during that time, it will display the menu and wait for input. If a hotkey associated with a menu entry is pressed, it will boot the associated menu entry immediately. If the timeout expires before either of these happens, it will boot the default entry. In the ‘countdown’ case, it will show a one-line indication of the remaining time.

```
https://www.gnu.org/software/grub/manual/grub/html_node/Simple-configuration.html

Reply:
Let's use 'menu' for now since that's the default if it's unset - and as discussed - let's change the default upstream so it's unset.

Reply:
Just my opinion, but I quite like the menu option. It gives a much longer window to edit boot options when we hit issues with booting. Particularly helpful on some of the more sluggish virtual consoles that drop inputs.